.clang-format

```diff
@@ -0,0 +1,9 @@
+---
+# Requires clang-format 10 or newer
+Language:        Cpp
```

Comment:
You can remove these three lines.

Add a license header.

Reply:
I mentioned this in the PR description. MacOS builds fail as a direct consequence of applying `clang-format`, and this is needed to make them succeed.

> The format moved the includes around, and so on MacOS `vpi_user.h` didn't have the typedef for `uint64_t`. I added `__APPLE__` to the `stdint.h` include for both VPI and VHPI headers.